sunkit_pyvista/version.py

```diff
@@ -5,4 +5,4 @@
     from setuptools_scm import get_version
     __version__ = get_version(root='..', relative_to=__file__)
 except Exception:
-    __version__ = '0.1.dev4+g7f5b6c0.d20210519'
+    __version__ = '0.1.dev17+g3b166a0.d20210526'
```

Comment:
This should be "0.0.0" or something from what I recall.

Reply:
Should this file even be here, or should it be gitignored? I can fix this in another PR either way.

Reply:
Hmm, I think this file should gitignore.